lightly/cli/upload_cli.py

```diff
@@ -74,10 +76,10 @@ def _upload_cli(cfg, is_cli_call=True) -> Union[str, None]:
             cli_api_args_wrong = True
         else:
             api_workflow_client = \
-                ApiWorkflowClient(token=token, dataset_id=dataset_id)
+                ApiWorkflowClient(token=token, dataset_id=dataset_id, dataset_creator=DatasetCreator.USER_PIP_LIGHTLY_MAGIC)
```

Comment:
could this logic also be applied for the tags which are created by lightly-magic (and other stuff lightly-magic does?)

Reply:
Two points I checked:
- Yes, if you upsize using the CLI, the tag creator will be the `USER_PIP`.
- Yes, you could set a different tag creator here. However, I am not sure if we already have one.

In any case, this belongs to a follow-up issue. However, I am not sure about its importance.

Reply:
yes its important as we want the pip users separate in our stats. Will follow up with an issue